src/runtime/config/configuration-qemu.toml.in

```diff
@@ -33,6 +33,31 @@ machine_type = "@MACHINETYPE@"
 # Default false
 # confidential_guest = true
 
+# Enable pre-attestation AMD SEV and SEV-ES confidential guests.
+# Applies only if confidential_guest is true.
+# (default: false)
+#guest_pre_attestation = true
+#
+# Guest owner proxy that handles remote attestation
+#guest_pre_attestation_proxy="localhost:50051"
+#
+# Keyset ID for injected secrets
+#guest_pre_attestation_keyset="KEYSET-1"
+#
+# GUID of injected secret
+# Key Broker Server for SEV(-ES) expects secrets with this guid
+# See https://github.com/confidential-containers/simple-kbs
+#guest_pre_attestation_secret_guid="0a46e24d-478c-4eb1-8696-113eeec3aa99"
+#
+# Type of injected secret
+#guest_pre_attestation_secret_type="bundle"
+#
+# SEV guest policy
+#sev_guest_policy=0
+
+# SEV certificate chain path 
+#sev_cert_chain="/opt/sev/cert_chain.cert"
```

Comment:
When using SEV, we would like the distros / the users to set at least the `sev_cert_chain`, maybe the other fields as well.

The best way we have to do so, is by having an option as part of the `Makefile` that we can set during the build time.
This allows then a distro to set it accordingly to the location where they'll ship the `cert_chain.cert` without having to `sed` the file, or even worse, without having to connect to each node to change this.

With this in mind, would be possible to add that for the options that you think that should be configurable?

Reply:
*configurable at the build time.

Reply:
The SEV cert_chain isn’t shipped by any distros at this time, but instead must be imported by the host admin (to a non-specific location) through a third-party tool like [sevctl](https://github.com/virtee/sevctl).  So, I believe the best we can do here is provide instructions for how to import `cert_chain.cert`

I don't think any other sev/attestation parameter should be configured at build time, for that matter.

Reply:
> The SEV cert_chain isn’t shipped by any distros at this time, but instead must be imported by the host admin (to a non-specific location) through a third-party tool like [sevctl](https://github.com/virtee/sevctl). So, I believe the best we can do here is provide instructions for how to import cert_chain.cert

Okay, I'd suggest then to make this a variable that could be set during the build time.
It'll help us with the Ci in the future.

> I don't think any other sev/attestation parameter should be configured at build time, for that matter.

Perfect, thanks!

Reply:
All set